pkg/cmd/create.go

```diff
@@ -164,9 +186,9 @@ func createAction(cmd *cobra.Command, args []string) error {
 	}
 	clusterStackPath := args[0]
 
-	if mode != stableMode && mode != hashMode {
+	if mode != stableMode && mode != hashMode && mode != customMode {
 		fmt.Println("The mode is ", mode)
-		return fmt.Errorf("mode is not supported please choose from - stable, hash")
+		return fmt.Errorf("mode is not supported please choose from - stable, hash, custom")
```

Comment:
I would add the variable "mode" into Errorf and remove the fmt.Println(). The Println looks like debug-code.